gamemode/modules/base/sv_interface.lua

```diff
@@ -67,6 +67,12 @@ DarkRP.retrievePlayerData = DarkRP.stub{
             description = "The function that receives the information.",
             type = "function",
             optional = false
+        },
+        {
+            name = "failure",
+            description = "The function that is called when the information cannot be retrieved.",
+            type = "function",
+            optional = false
```

Comment:
the "failed" argument was your choice in retrievePlayerData @FPtje and is
also used for asynchronous MySQLite database failure checking which is why
I put it in. not sure why he's referring to it has a "check" however

On Thu, Mar 17, 2016 at 1:22 PM, Falco Peijnenburg <notifications@github.com
> wrote:

> In gamemode/modules/base/sv_interface.lua
> <https://github.com/FPtje/DarkRP/pull/2368#discussion_r56501852>:
>
> > @@ -67,6 +67,12 @@ DarkRP.retrievePlayerData = DarkRP.stub{
> >              description = "The function that receives the information.",
> >              type = "function",
> >              optional = false
> > +        },
> > +        {
> > +            name = "failure",
> > +            description = "The function that is called when the information cannot be retrieved.",
> > +            type = "function",
> > +            optional = false
>
> why do you refer to it as "this check"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FPtje/DarkRP/pull/2368/files/0dbb2b2959e7a8083d2cd74b89c97a5cb07bb4ba#r56501852>
>